### ISSUE TITLE
Files in repository (Maybe?) not applied to Update or CG archive

### ISSUE THREAD

**Issue by drojf** [07th-mod/higurashi-console-arcs#596]
There are currently two folders in the main repo which are not included in the Update archive: the 'CG' archive, and the 'OGSprites' archive.

Was discussing with drdiablo and was wondering if they were already applied to the main CG archive (or some other archive).

I've cross checked and here's what I found:

## OGSprites

The OGSprites in the repository is not in the correct folder structure - the spritesare directly in the OGSprites folder, but in the CG archive the images should be placed in a `sprites` subdirectory. When you compare the images (compensating for the wrong folder structure), some of the sprites are different (the expressions look different)

- ha1_odoroki_0.png
- ta3_human_0.png
- ta3_iradachi_0.png
- ta3_sakebi_0.png

ta3_iradachi_0.png - LEFT: repo RIGHT: CG Archive
![image](https://user-images.githubusercontent.com/1249449/206820972-2c662331-5acc-4279-9965-0cd73d36f404.png)

## CG folder

The CG folder is in the correct folder structure, but the following images differ:

- tips/omot002.png
- tips/omot003.png (these are swapped around, perhaps you fixed it in the CG archive at some point)
- scenario/connecting/background.png
- scenario/omote/background.png
- scenario/ura/background.png (the CG archive has a more contrasty version)

tips/omot002.png - LEFT: repo RIGHT: CG Archive
![image](https://user-images.githubusercontent.com/1249449/206820827-9e62b18a-902b-4446-845d-ba514d346478.png)

scenario/connecting/background.png - LEFT: repo RIGHT: CG Archive

![image](https://user-images.githubusercontent.com/1249449/206820803-f139cf5b-eab0-471f-b513-d4e7a4e54302.png)

I also cross-checked against a very old version of the graphics archive from dec 2020 (in case these changes are stuff you fixed recently), but the changes are the same

**Comment by DoctorDiablo** [comment]
I doublechecked and the ones in the CG folder have been corrected in the latest graphics archive. The swapped order on those images was a bug fix that got applied at some point, and the high contrast version makes it consistent with similar backgrounds. I think those can be deleted from github.

The ones in the OGSprites folder are a little more complicated... Normally we just use the OG sprites from the steam version as-is, but we couldn't do that for Miotsukushi because we released Mio before Matsuribayashi ever came out.  So we had to special make all these extra poses by pulling the images from the old old higu release. I think as long as these poses exist in the Matsuribayashi or Rei archives we can just copy the files from there and ignore the entries in git, because those should be correct and consistent with our other releases.

**Comment by drojf** [comment]
OK, I'll try to source the sprites from Matsu/Rei or wherever I can find them, then put them in the Console update archive (https://07th-mod.com/rikachama/og-dll-asset-update/console-og-dll-asset-update.7z)

I'll only put them in the update archive if they don't already match the ones in the CG archive.

ha1_odoroki_0.png
ha1_shinken_0.png
ha3a_au_0.png
ha3a_warai_0.png
ha3b_warai_0.png
ta1_human_0.png
ta1_iradachi_0.png
ta1_sakebi_0.png
ta3_hatena_0.png
ta3_human_0.png
ta3_iradachi_0.png
ta3_sakebi_0.png
ta4_def_0.png
ta4_niyari_0.png
ta4_sakebi_0.png
tomi2_ikari_0.png
tomi2_shinken_0.png

**Comment by drojf** [comment]
I tried checking, but it looks like many of the sprites in the above list don't exist in matsu or rei (grey in the below picture). Out of the ones which exist, only two are different, but due to how I generated the OG Sprites, probably the ones you already have in the Console arcs are the more correct ones for Console.

![image](https://user-images.githubusercontent.com/1249449/206882295-0b7d9854-6cb4-4ff4-acf1-1bbd0d66e96a.png)

For this reason, I think I shouldn't make any changes to the OGSprites in the CG archive.

So in summary, I think we can remove the files from the repo - what is in the CG archive is correct.

**Comment by drojf** [comment]
I'll go ahead and remove the images from the repo now - below is an archive of the 4 images which are different from the existing archive:

[console-repo-og-sprites-differences-only.zip](https://github.com/07th-mod/higurashi-console-arcs/files/10252858/console-repo-og-sprites-differences-only.zip)

And a complete list of files in the OGSprites folder:

```
ha1_odoroki_0.png
ha1_shinken_0.png
ha3a_au_0.png
ha3a_warai_0.png
ha3b_warai_0.png
ta1_human_0.png
ta1_iradachi_0.png
ta1_sakebi_0.png
ta3_hatena_0.png
ta3_human_0.png
ta3_iradachi_0.png
ta3_sakebi_0.png
ta4_def_0.png
ta4_niyari_0.png
ta4_sakebi_0.png
tomi2_ikari_0.png
tomi2_shinken_0.png
```
